src/App.js

```diff
@@ -1,10 +1,26 @@
 import React from "react";
-import MainPage from "./components/mainpage/componens.mainpage.mainpage";
+import "./App.scss";
+import { Switch, Route } from "react-router";
+import SideBar from "./components/sidebar/sidebar.components";
+import Chatroom from "./components/chatroom/chatroom.components";
+import Profile from "./components/profile/profile.components";
+import Other from "./components/other/other.components";
 
 function App() {
   return (
-    <div>
-      <MainPage />
+    <div className="App">
+      <SideBar></SideBar>
```

Comment:
안에 감싸는 게 없다면 `self-closing tag`를 사용하는 게 더 깔끔해 보일 것 같습니다!
```suggestion
      <SideBar/r>
```